src/wp-admin/options-general.php

```diff
@@ -506,6 +506,36 @@
 ?>
 </select></td>
 </tr>
+
+
+<tr>
+<th scope="row"><label for="blocks_compatibility_level"><?php _e( 'Blocks Compatibility' ); ?></label></th>
+<td><select name="blocks_compatibility_level" id="blocks_compatibility_level">
+<?php
+$blocks_compatibility_level = get_option( 'blocks_compatibility_level', 1 );
+
+$blocks_compatibility_level_desc = array(
+	_x( 'Off', 'Block Compatibility' ),
+	_x( 'On', 'Block Compatibility' ),
+	_x( 'Troubleshooting', 'Block Compatibility' ),
+);
+
+for ( $index = 0; $index <= 2; $index++ ) {
+	echo "\n\t<option value='" . esc_attr( $index ) . "' " . selected( $blocks_compatibility_level, $index, false ) . '>'
+	. esc_html( $blocks_compatibility_level_desc[ $index ] ) . '</option>';
+}
+?>
+</select>
+<p class="description" id="home-description">
+<?php _e( 'ClassicPress is incompatible with the block editor. When blocks compatibility is turned on (default), it allows more plugins and themes to work but block-related features will not work. ' ); ?>
+<a href="https://docs.classicpress.net/user-guides/using-classicpress/settings-general-screen/#blocks-compatibility" target="_blank" rel="noopener noreferrer">
```

Comment:
I didn't notice this before. We should remove `target="_blank"` for accessibility and `rel="noopener noreferrer"` to keep it consistent with all other similar links in admin. Here's an example lower down the page:
https://github.com/ClassicPress/ClassicPress-v2/blob/4ee146a6da6d4c35a407126dca5eea989d9fe3cf/src/wp-admin/options-general.php#L488

While we're at it, custom image link should also be fixed in this file:
https://github.com/ClassicPress/ClassicPress-v2/blob/4ee146a6da6d4c35a407126dca5eea989d9fe3cf/src/wp-admin/options-general.php#L179-L181

```
<p class="description" id="login_custom_image-description">
<?php _e( 'If you choose an image here and enable this option, then that image will be shown at the top of the login page instead of the ClassicPress logo. ' ); ?>
<a href="https://link.classicpress.net/docs/custom-login-image">
<?php _e( 'Learn more.' ); ?>
</a>
</p>
```